policybot/mgrs/syncmgr/mgr.go

```diff
@@ -211,7 +211,7 @@ func (sm *SyncMgr) Sync(context context.Context, flags FilterFlags, dryRun bool)
 	}
 
 	if flags&TestResults != 0 {
-		if err := ss.handleTestResults(); err != nil {
```

Comment:
This needs to be removed for now, until we have added the Feature Coverage measurement...